websites/S/Skribbl.io/metadata.json

```
@@ -17,7 +21,7 @@
 		"nl": "Skribbl.io is een gratis multiplayer teken- en raadspel. Teken en raad woorden met je vrienden en mensen van over de hele wereld! Scoor de meeste punten en wees de winnaar!"
 	},
 	"url": "skribbl.io",
-	"version": "1.1.22",
+	"version": "1.2.0",
 	"logo": "https://i.imgur.com/au4OO2A.jpg",
```

Let's change it to 512x512
```suggestion
	"logo": "https://i.imgur.com/EYcWQmj.png",
```